tests_require_fixes/utils/load_one_data.py

```
@@ -0,0 +1,118 @@
+from typing import Callable
+
+
+def dump_one(data: dict, file_name: str) -> None:
+    diff_data_path = Path(__file__).parent / "diff_data"
+    diff_data_path.mkdir(exist_ok=True, parents=True)
+    file_path = diff_data_path / f"{file_name}.json"
+
+    try:
+        with open(file_path, "w", encoding="utf-8") as f:
+            json.dump(data, f, ensure_ascii=False, indent=4)
+    except Exception as e:
```

Ruff (0.14.14)

Do not catch blind exception: `Exception`

(BLE001)